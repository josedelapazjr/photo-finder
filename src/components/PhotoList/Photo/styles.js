export default {
  root: {
    padding: 10,
    cursor: 'pointer',
  }
}