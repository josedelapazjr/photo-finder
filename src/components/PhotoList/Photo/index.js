import React from 'react';
import injectSheet from 'react-jss';
import styles from './styles';

const Photo = ({photo, classes, handleViewDetails}) => (
  <div className={classes.root} onClick={() => handleViewDetails(photo.id)}>
    {photo 
      ? <img src={photo.urls ? photo.urls.small : photo.cover_photo.urls.small} title={photo.title}></img>
      : null
    }
  </div>
);

export default injectSheet(styles)(Photo);