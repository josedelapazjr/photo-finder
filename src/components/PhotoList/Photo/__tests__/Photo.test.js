import React from 'react';
import renderer from 'react-test-renderer';

import Photo from '../../Photo';

it('renders Photo component correctly', () => {
  const tree = renderer.create(<Photo />).toJSON();
  expect(tree).toMatchSnapshot();
});