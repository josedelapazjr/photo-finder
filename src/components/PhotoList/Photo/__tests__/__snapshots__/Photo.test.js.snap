// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Photo component correctly 1`] = `
<div
  className="Photo-root-0-1-1"
  onClick={[Function]}
/>
`;
